### DISCUSSION TITLE
Is there a correlation between APM and Agent version？

### DISCUSSION THREAD

**Discussion by luojie233** [apache/skywalking#10959] [General]
for example, can i use APM v8.8.0 with Java Agent v8.16.0?

**Comment by wu-sheng** [comment]
v8 is EOL. We don't maintain anything about it.
Docs in the each release tar includes all docs back then.

**Reply by luojie233** [reply]
OK. Thank you!

**Reply by daifang-code** [reply]
can you tell me in which doc？I didn't find where exactly.
